docs/Integration Test Guide.md

```diff
@@ -0,0 +1,73 @@
+# Integration test guide
+
+## Running integration tests
+
+The integration tests can be run in two different ways: locally with `go test` or via a GitHub Workflow.
+
+### Locally
+For running the integration tests locally, you have to clone the repository and install the required Go version.
+Additionally, the following environment variables have to be defined: 
+* `KEPTN_ENDPOINT` - The API endpoint from Keptn (e.g.: `http://10.201.4.2/api`)
+* `KEPTN_API_TOKEN` - The auth token for Keptn
+* `JES_E2E_TEST` - Must be set to `true` to enable the integration tests
+* `JES_NAMESPACE` - The namespace in which the job-executor-service was installed (e.g.: `keptn-jes`)
+
+After these environment variables have been defined, the test can be run with `go test -v test/e2e/*` or you can
+pick specific tests by defining a filter via the `-run` parameter:
+
+```bash
+go test -v -run "^\QTestEnvironmentVariables\E$"
+```
+
+*Note: Be aware that running `go test ./...` while `JES_E2E_TEST` is set to `true` will also run the integration tests!*
```

Comment:
not sure if we want this behaviour. Is it not possible to do some generic expression using `go test -run`?

Reply:
If no Keptn Endpoint is specified all test will fail with an error